bigquery/README.md

```diff
@@ -4,3 +4,55 @@ This notifier pushes build data to a BigQuery instance.
 
 *Alpha Feature - Under Development*
 
+## Accessing build insights with SQL queries through the BigQuery CLI:
+
+```bash
+$ bq query '<SQL QUERY>'
```

Comment:
Looking at the BQ CLI docs, you can supply `--nouse_legacy_sql` instead of writing the BQ rc file.